common/store.go

```diff
@@ -68,11 +68,24 @@ func StringToBackendType(s string) BackendType {
 	}
 }
 
+type CertVerificationOpts struct {
+	// L1 block number at which the cert was included in the rollup batcher inbox.
+	// This is optional, and should be set to 0 to mean to skip the RBN recency check.
+	// It is impossible for a batch inbox tx to have been included in the genesis block,
+	// so we are free to give this special meaning to the zero value.
+	//
+	// Used to determine the validity of the eigenDA batch.
+	// The eigenDA cert contains a reference block number (RBN) which is used
+	// to lookup the stake of the eigenda operators before verifying signature thresholds.
+	// The rollup commitment containing the eigenDA cert is only valid if it was included
+	// within a certain number of blocks after the RBN.
+	// validity condition is: certRBN < L1InclusionBlockNum < RBN + RBNRecencyWindowSize
```

Comment:
I think you've misstated the validity case, it ought to read:

`certRBN < L1InclusionBlockNum <= certRBN + RBNRecencyWindowSize`

Personally, I try to pick either the "valid" or the "not valid" perspective, and make _every_ doc and piece of code approach from that same perspective. As it currently stands, some of your docs take the "valid" perspective, and others take the "invalid" perspective. It's really easy to introduce these off-by-one errors, trying to convert between the positive and negative case in your head.

Reply:
good catch: [f0e1e9a](https://github.com/Layr-Labs/eigenda-proxy/pull/393/commits/f0e1e9a5f0a87d31ebd61ae4c73767598eeef197)
Agree with you.. I think the main issue is just that I have this (in)validity condition restated in way too many places. I tried to use the doc symlinks but it doesn't work as well as I would have liked. I'm not sure how to best do this, as I still think its good to have the same validity condition stated at least on the flag, config, and main verifying function logic.

I think everywhere left is using the validity perspective now? But let me know if not.

Reply:
I agree, it's worth restating the validity criteria, even if it's duplicated

There are still some places in the proxy where you state the "stale" case, instead of the validity case:

- `.env` files
- `client_config_v2.go`
- `config/v2/eigendaflags/cli.go`

Reply:
[48be85c](https://github.com/Layr-Labs/eigenda-proxy/pull/393/commits/48be85cdd61398e71660fe8c24dbfab1ac90e38a)